model/riscv_vmem_ptw.sail

```diff
@@ -60,7 +60,7 @@ function translationException(a : AccessType(ext_access_type),
     (Read(_), _)                 => E_Load_Page_Fault(),
     (Write(_), PTW_Access())     => E_SAMO_Access_Fault(),
     (Write(_), _)                => E_SAMO_Page_Fault(),
-    (Execute(), PTW_Access())    => E_Fetch_Access_Fault(),
-    (Execute(), _)               => E_Fetch_Page_Fault()
+    (InstructionFetch(), PTW_Access())    => E_Fetch_Access_Fault(),
```

Comment:
Probably wouldn't update the alignment of the lines above but at least remove the extra spaces before `=>`.